### PR TITLE
No Spoofing Protection on Email Domain

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/mail_server_misconfiguration/no_spoofing_protection_on_email_domain/template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+1. Use dig or nslookup to request details for SPF/DKIM on the domain:
+
+```bash
+dig TXT 
+```
+
+1. Observe in the response that the SPF/DKIM is missing or misconfigured:
+
+{{response}}
+
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/mail_server_misconfiguration/no_spoofing_protection_on_email_domain/template.md:22]: There should also be DMARC here, as there is 0 spoofing protections.

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 change then LGTM

File: submissions/description/server_security_misconfiguration/mail_server_misconfiguration/no_spoofing_protection_on_email_domain/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/mail_server_misconfiguration/no_spoofing_protection_on_email_domain/template.md:22]: Thank you for this pick-up mate!

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**